misc/python/materialize/cli/helm_chart_version_bump.py

```diff
@@ -19,14 +19,16 @@
 def main() -> int:
     parser = argparse.ArgumentParser(
         prog="helm-chart-version-bump",
-        description="Bump environmentd/orchestratord versions in helm chart.",
+        description="Bump environmentd (appVersion) and helm-chart versions in helm chart.",
     )
     parser.add_argument(
         "--helm-chart-version",
         type=str,
         help="Helm-chart version to bump to, no change if not set.",
     )
-    parser.add_argument("version", type=str, help="Materialize version to bump to.")
+    parser.add_argument(
+        "environmentd-version", type=str, help="environmentd version to bump to."
```

Comment:
Oh, hey @def-  
With this variable name change, shouldn't we update lines 42 and 48 to use the new variable name?

Reply:
oops, of course!